frontend/src/metabase/css/dashboard.css

```diff
@@ -3,6 +3,7 @@
 }
 .Dashboard {
   background-color: #f9fbfc;
+  height: 100vh;
```

Comment:
@kdoh I did this because the background color of dashboards wasn't extending the full height of the browser window (even on master), and height: 100% wasn't working (presumably because of a parent container's height?). Got a better idea?

Reply:
I think we might want to use our `withBackground` util in this case. It sets the body bg color for pages it's applied to which means we won't have to worry about the height or set a bg color in that css file. I can look at that.

Reply:
Cool. We might want to apply the same to the sidebar with its slightly darker color while we're at it. Thought that I'd fixed it in a previous commit by giving it full-height but it doesn't appear to be working in cases  where the height of the dashboard exceeds the height of the sidebar.